lib/getMsMd.m

```diff
@@ -0,0 +1,49 @@
+function [Ms, Md] = getMsMd(units, M, DIM)
+    % The below code was taken from the spm_mip_ui.m file in SPM12. 
```

Comment:
In the spm_mip_ui lines 259:261 cf:https://github.com/neurodebian/spm12/blob/master/spm_mip_ui.m#L259-L261 a call is made to spm_mip but it required the transformations first. I believe either you or @nicholst said that the code from the emails for creating XYZ, Tth and Z was derived from the spm_mip_ui code - therefore I thought the same transformations that this code applies would have to be applied here.